recipes-extended/jailhouse/jailhouse-imx_git.bb

```diff
@@ -91,6 +91,7 @@ PACKAGE_BEFORE_PN = "pyjailhouse"
 FILES:${PN} += "${nonarch_base_libdir}/firmware ${libexecdir} ${sbindir} ${JH_DATADIR}"
 # Remove libdir/* appended by setuptools3-base.bbclass for module split to work correctly
 FILES:${PN}:remove = "${libdir}/*"
+FILES:${PN}-dev += "${RECIPE_SYSROOT_NATIVE}${PYTHON_SITEPACKAGES_DIR}/*"
```

Comment:
`inherit python3targetconfig` might help

Reply:
> `inherit python3targetconfig` might help

@kraj Thanks a lot, I'll try it tomorrow morning :)

Reply:
@kraj Unfortunately, it doesn't help.

```
ERROR: jailhouse-imx-git-r0 do_package: QA Issue: jailhouse-imx: Files/directories were installed but not shipped in any package:
  /home
  /home/cryo
  /home/cryo/work
  /home/cryo/work/freescale
  /home/cryo/work/freescale/build-fslc-framebuffer
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/direct_url.json
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/COPYING
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/WHEEL
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/top_level.txt
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/RECORD
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/METADATA
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/INSTALLER
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse-0.12.dist-info/REQUESTED
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/extendedenum.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/config_parser.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/pci_defs.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/cell.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__init__.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/sysfs_parser.py
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/sysfs_parser.cpython-311.pyc
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/cell.cpython-311.pyc
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/config_parser.cpython-311.pyc
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/__init__.cpython-311.pyc
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/extendedenum.cpython-311.pyc
  /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/recipe-sysroot-native/usr/lib/python3.11/site-packages/pyjailhouse/__pycache__/pci_defs.cpython-311.pyc
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
jailhouse-imx: 38 installed and not shipped files. [installed-vs-shipped]
ERROR: jailhouse-imx-git-r0 do_package: Fatal QA errors were found, failing task.
ERROR: Logfile of failure stored in: /home/cryo/work/freescale/build-fslc-framebuffer/tmp/work/imx8mq_lpddr4_wevk-fslc-linux/jailhouse-imx/git/temp/log.do_package.9768
ERROR: Task (/home/cryo/work/freescale/sources/meta-freescale/recipes-extended/jailhouse/jailhouse-imx_git.bb:do_package) failed with exit code '1'
```

Reply:
@kraj @otavio We see a similar issue in our project LmP with the latest oe.
Trying to find out the root cause may take a lot of time, so I'd suggest using the hack for now and scheduling an investigation after NY.

Reply:
packaging native sysroot elements seems quite wrong to me. What happens if you do not package them.

Reply:
@MrCry0, have you made any progress on this?

Reply:
@otavio I haven't found another solution and will be unavailable for 5-6 weeks.
@quaresmajose said he may have time to find a more generic solution or fix it on the OE level.

Reply:
https://github.com/Freescale/meta-freescale/pull/1738 should fix this issue so the line below can be removed
```
FILES:${PN}-dev += "${RECIPE_SYSROOT_NATIVE}${PYTHON_SITEPACKAGES_DIR}/*"
```